src/main/java/dev/socketmods/socketnukes/explosion/ExplosionProperties.java

```diff
@@ -0,0 +1,41 @@
+package dev.socketmods.socketnukes.explosion;
+
+import net.minecraft.particles.IParticleData;
+import net.minecraft.util.SoundEvent;
+
+/**
+ * Stores all the needed metadata for explosions, for carrying around information like the sound, particle,
+ * whether it creates fire, etc etc.
+ *
+ * @author Citrine
+ */
+public class ExplosionProperties {
```

Comment:
Make the fields final and remove the `= false`? We don't modify this anywhere else.